crates/consensus-logic/src/worker.rs

```diff
@@ -149,12 +142,12 @@ fn process_msg<D: Database, E: ExecEngineCtl>(
                 warn!(%missed_ev_cnt, "applying missed sync events");
                 for ev_idx in next_exp_idx..*idx {
                     trace!(%ev_idx, "running missed sync event");
-                    handle_sync_event(state, engine, ev_idx, status_rx.clone(), fcm_msg_tx)?;
+                    handle_sync_event(state, engine, ev_idx, status_rx.clone())?;
```

Comment:
Can probably avoid this clone here, since we're in here making changes.  Pass more by-ref.